ui/src/adapters/api/credentials.ts

```diff
@@ -76,6 +79,92 @@ const resultOKCredentials = getStrictParser<ResultOK<Credential[]>>()(
   })
 );
 
+export const linkStatusParser = getStrictParser<LinkStatus>()(
+  z.union([z.literal("active"), z.literal("inactive"), z.literal("exceeded")])
+);
+
+const linkAttributes = getStrictParser<LinkAttributes>()(
+  z.object({
+    name: z.string(),
+    value: z.string(),
+  })
+);
+
+export const link = getStrictParser<Link>()(
+  z.object({
+    active: z.boolean(),
+    attributes: z.array(linkAttributes),
+    expiration: z.coerce.date().optional(),
+    id: z.string(),
+    issuedClaims: z.number(),
+    maxIssuance: z.number().optional(),
+    schemaType: z.string(),
+    status: linkStatusParser,
+  })
+);
+
+export async function getLinks({
+  env,
+  params: { query, status },
+  signal,
+}: {
+  env: Env;
+  params: {
+    query?: string;
+    status?: LinkStatus;
+  };
+  signal: AbortSignal;
+}): Promise<APIResponse<Link[]>> {
+  try {
+    const response = await axios<Link[]>({
```

Comment:
This makes it clear "quick reviews" do not work. This should have never been merged.